src/components/templates/Menu/MenuTemplate.jsx

```diff
@@ -56,11 +52,7 @@ function MenuTemplate() {
           </div>
         </div>
         <div className={css.menu__buttons}>
-          <MenuButtonsOrganism
-            handleClick={handleClick}
-            handleChange={handleChange}
-            text={buttonsText}
-          />
+          <MenuButtonsOrganism handleChange={handleChange} text={buttonsText} />
```

Comment:
Ok, but here u still passing handleChange from up to down component.

Reply:
@TN4HPEL3 I've fixed that.